pantheon/src/test/java/tech/pegasys/pantheon/cli/PantheonCommandTest.java

```diff
@@ -351,6 +386,58 @@ public void tomlThatConfiguresEverythingExceptPermissioning() throws IOException
     assertThat(options.stream().map(CommandLine.Model.OptionSpec::longestName)).isEmpty();
   }
 
+  @Test
+  public void tomlThatConfiguresEverythingPlusPermissioning() throws IOException {
+    assumeTrue(isFullInstantiation());
+
+    // Load a TOML that configures literally everything (except permissioning)
```

Comment:
Could bits of the `tomlThatConfiguresEverythingExceptPermissioning` method be extracted to avoid dupe here?

Reply:
actually examining how the spec options work, this method doesn't test anything extra. removed.